src/chainparams.cpp

```diff
@@ -39,7 +39,9 @@ class CMainParams : public CChainParams {
         // TODO generate harder genesis block
         //consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.powLimit = uint256S("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
-        consensus.nPowTargetTimespan = 14 * 24 * 60 * 60; // two weeks
+        consensus.nPowMaxAdjustDown = 16; // 16% adjustment down
+        consensus.nPowMaxAdjustUp = 8; // 8% adjustment up
+        consensus.nPowTargetTimespan = 17 * 10 * 60; // 17 blocks
         consensus.nPowTargetSpacing = 10 * 60;
```

Comment:
Oh never mind, the target spacing is right here. When we're changing this constant we should notice the one above :)